src/CMakeLists.txt

```diff
@@ -540,7 +559,7 @@ find_package(Pugixml 1.2 REQUIRED)
 include_directories(SYSTEM ${Pugixml_INCLUDE_DIRS})
 list(APPEND LIBS ${Pugixml_LIBRARIES})
 
-if(NOT SOURCE_PACKAGE AND NOT APPLE)
+if(NOT SOURCE_PACKAGE AND NOT APPLE AND NOT WIN32)
```

Comment:
> Where is the -Wshadow disabled? If I remove the -Werror definition from CmakeList.txt, rawspeed gives me these shadow warnings still. I see -Wshadow ignore pragmas in the code elsewehere (like masks.c), but for Rawspeed I could not find.

So are these warnings or errors? Please be specific.
They are supposed to be warnings.